doc/05-common-errors.md

```diff
@@ -2,6 +2,24 @@
 title: Common errors
 ---
 
+# `c_poll` permission denied
+
+**TL;DR**: The error is harmless.
+
+**Long version**:
+
+Sometimes, it is possible that the runtime can throw an error that looks more or less like this:
```

Comment:
Nitpick: I do not think that this should not be named "throw". This implies that we get an exception in Haskell land, at least to me.

I'd say "print" here.